qemu/tests/enable_scatter_windows.py

```diff
@@ -172,28 +176,44 @@ def _get_driver_version(session):
         lambda: _is_process_finished(session, autoit_name), timeout, 20, 3
     )
 
-    error_context.context("Check if wireshark is installed", test.log.info)
-    check_installed_cmd = params.get("check_installed_cmd")
-    check_result = session.cmd_output(check_installed_cmd)
-    if "tshark" not in check_result:
-        error_context.context("Install wireshark", test.log.info)
-        install_wireshark_cmd = params.get("install_wireshark_cmd")
-        install_wireshark_cmd = utils_misc.set_winutils_letter(
-            session, install_wireshark_cmd
-        )
-        status, output = session.cmd_status_output(
-            install_wireshark_cmd, timeout=timeout
+    for tool in ["windump", "tshark"]:
+        check_cmd = params["check_%s_installed_cmd" % tool]
+        install_cmd = utils_misc.set_winutils_letter(
+            session, params["%s_install_cmd" % tool]
         )
-        if status:
-            test.error(
-                "Failed to install wireshark, status=%s, output=%s" % (status, output)
+        process_name = tool
```

Comment:
Why do we need this renaming? You might directly have `for process_name in ["windump", "tshark"]` if you feel like name it this way.